src/crc8.rs

```diff
@@ -47,13 +47,17 @@ impl Crc<u8> {
     }
 
     pub const fn digest(&self) -> Digest<u8> {
-        Digest::new(self)
+        let initial = self.init();
+        Digest::new(self, initial)
+    }
+
+    pub const fn digest_with_initial(&self, initial: u8) -> Digest<u8> {
```

Comment:
Should `initial` be reversed if `refin` is set to true?

Reply:
Yes, good catch. I updated the PR accordingly.